cms/models.py

```diff
@@ -35,14 +36,17 @@ def get_context(self, request):
             "host": webpack_dev_server_host(request)
         }
 
+        username = None
+        if not request.user.is_anonymous():
+            username = request.user.social_auth.get(provider=EdxOrgOAuth2.name).uid
```

Comment:
Yes. Fixing it here is not difficult but I'm not sure where we want staff users to be able to go